SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -22,9 +22,19 @@ struct FPendingRPCParams
 	SpatialGDK::RPCPayload Payload;
 };
 
-class FRPCContainer
+class SPATIALGDK_API FRPCContainer
 {
 public:
+	FRPCContainer() = default;
+
+	FRPCContainer(const FRPCContainer&) = delete;
```

Comment:
Mostly a nit - would prefer if we keep the style on this kind of thing consistent. I think the more commonly used one is 
// Moveable, not copyable.
Copy constructor
Move constructor
Copy assignment
Move assignment

Reply:
Fixed.